### PR TITLE
Docs(bp-kits):updated arc42 view

### PR DESCRIPTION
**Pull request by Unsharm**
## Description
In this pull request, we have updated the Architecture view of the Business Partner KIT.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md [hunk 1]
```diff
@@ -224,92 +216,8 @@ Due to a transmission phase there are two concepts of Business Partner Upload Mo
 
 -Currently there is no SME Application available
```
Comment (SujitMBRDI) [inline docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:217]: Can you please add space here between '-' and 'Currently there is no SME Application available'

File: docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md [hunk 2]
```diff
@@ -592,6 +486,198 @@ UpdateLayoutConfig($c4ShapeInRow="3", $c4BoundaryInRow="1")
 
 ## Crosscutting Concepts
 
+### Authentication & Autorization
+
+### Roles, Rights, Permissions
```
Comment (SujitMBRDI) [inline docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:491]: Please make relevant sub headings here, currently `Roles, Rights, Permissions` should appear in same precision of heading `Authentication & Autorization`

File: docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md [hunk 3]
```diff
@@ -614,21 +700,369 @@ In addition to the Spring standard logs the BPDM applications keep a log of the
...
+
+- No EDC enabled communication is needed between the bpdm pool and the backend service that processes the data.
+![Internal web application that only visualizes data based on gate data](@site/static/img/kit-icons/edc-usage-2-2.drawio.svg)
 
 ## Quality Requirements
 
 - see: [Quality Goals](#quality-goals-quality-goals)
 
-## Risks and Technical Debts
+## Risks and Technical Debts
```
Comment (SujitMBRDI) [inline docs-kits/kits/Business Partner Kit/Documentation BPDM/bpdm_arc42.md:1065]: Please remove tail space

**Review by SujitMBRDI — CHANGES_REQUESTED** [review]
Changes looks good to me, suggested some minor changes.

**Review by SujitMBRDI — APPROVED** [review]

**Review by maximilianong — APPROVED** [review]

**Pull request merged**